cmd/relay/commands/claim.go

```diff
@@ -0,0 +1,139 @@
+package commands
+
+import (
+	"bufio"
+	"encoding/csv"
+	"fmt"
+	"io"
+	"os"
+
+	cfg "github.com/iden3/go-iden3/cmd/relay/config"
+	common3 "github.com/iden3/go-iden3/common"
+	"github.com/iden3/go-iden3/core"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var ClaimCommands = []cli.Command{
+	{
+		Name:  "claim",
+		Usage: "claim add",
+		Subcommands: []cli.Command{{
+			Name:   "add",
+			Usage:  "claim add",
+			Action: cmdAddClaim,
+		}},
+	},
+	{
+		Name:  "claims",
+		Usage: "claims import from file",
+		Subcommands: []cli.Command{{
+			Name:   "fromfile",
+			Usage:  "import claims from file",
+			Action: cmdAddClaimsFromFile,
+		}},
+	},
+}
+
+func cmdAddClaim(c *cli.Context) error {
+	if err := cfg.MustRead(c); err != nil {
+		return err
+	}
+
+	ks, acc := cfg.LoadKeyStore()
+	client := cfg.LoadWeb3(ks, &acc)
+	storage := cfg.LoadStorage()
+	mt := cfg.LoadMerkele(storage)
+
+	rootservice := cfg.LoadRootsService(client)
+	claimservice := cfg.LoadClaimService(mt, rootservice, ks, acc)
+
+	indexData := c.Args().Get(0)
+	outData := c.Args().Get(0)
```

Comment:
Is indexData == outData? (both use Get(0))

Maybe it should be `outData := c.Args().Get(1)`?

Reply:
done